android-sdk/src/main/AndroidManifest.xml

```diff
@@ -8,23 +8,24 @@
     <application
         android:label="@string/app_name"
         android:supportsRtl="true">
-        <activity
-            android:name="org.sagebionetworks.bridge.android.AuthManagementActivity"
-            android:launchMode="singleTask">
-            <tools:validation
-                testUrl="https://webservices.sagebridge.org/mobile/startSession.html?email=test1%40sagebase.org&amp;study=study-id&amp;token=TOKEN"/>
+        <!-- Implement this activity for email-only sign in -->
```

Comment:
By defining this in the library's manifest it makes it impossible to override.  Usually you let the app define, and put it in the sdk docs that you need to.